### PR TITLE
Feature/optional header

### LINKED ISSUE #941: Remove the "Volgende experiment" button on Dashboard
**Is your feature request related to a problem? Please describe.**
I am not sure what does this "volgende experiment" button do, is there a specific reason that this button appeared in the Dashboard? :-)

**Describe the solution you'd like**
Remove the button if it is an accidental feature that is designed for the general dashboard but not the congo version.

### PR DESCRIPTION
**Pull request by BeritJanssen**
Close #941 by adding a configurable header to the theme. `ExperimentCollectionDashboard` will display a `<Header>` component if a theme with header is set.

Something that's now inconsistent with other views, but perhaps more consistent with json guidelines, is that the dicts sent to the frontend for `ExperimentCollection` objects will be in camel case, rather than snake case.

### CODE REVIEW

**Review by drikusroor — CHANGES_REQUESTED** [review]
I'm a bit concerned about both the `ThemeConfig` and the `DefaultPage` being somewhere in between something generic and something `Experiment(Collection)` oriented, though I'm not sure what the solution should be. Secondly, I feel like the frontend hasn't been finished properly: The experiment tiles seem to be vertically stacked and the other theme properties haven't been implemented. As for the latter, I think you could just copy + paste the font loader & background image configuration from the Experiment or perhaps even use this opportunity to abstract the font loader & background image to one single page/layout.

File: backend/experiment/serializers.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def serialize_experiment_collection(
         'description': experiment_collection.description,
         'consent': consent,
         'about_content': about_content,
+        'theme_id': experiment_collection.theme_config.pk if experiment_collection.theme_config else None
```
[the remark this answers is not in the document]
Comment (BeritJanssen) [reply backend/experiment/serializers.py:37]: I did this based on earlier feedback in which you seemed to be concerned about mixing different models in one serialization.

File: backend/theme/serializers.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        'privacy': footer.privacy
+    }
+
+
+def serialize_header(header: HeaderConfig) -> dict:
+    return {
+        'next_experiment_button_text': _('Next experiment'),
+        'about_button_text': _('About us'),
+        'show_score': header.show_score
+    }
```
[the remark this answers is not in the document]
Comment (BeritJanssen) [reply backend/theme/serializers.py:24]: Indeed, with #928 as a proposed course of action, I think the link of `Experiment` to `ThemeConfig` can be removed eventually.

File: frontend/src/components/Page/DefaultPage.jsx [hunk 3]
```diff
@@ -1,12 +1,36 @@
...
+import React, { useEffect, useState } from "react";
+import useBoundStore from "../../util/stores";
+
 import Page from "./Page";
 import AppBar from "../AppBar/AppBar";
+import Header from "../Header/Header";
 
 // DefaultPage is a Page with an AppBar and a width-restricted container for content
-const DefaultPage = ({ className, title, logoClickConfirm, children }) => {
+const DefaultPage = ({ className, title, logoClickConfirm, collectionSlug='', nextExperimentSlug='', children }) => {
```
[the remark this answers is not in the document]
Comment (BeritJanssen) [reply frontend/src/components/Page/DefaultPage.jsx:9]: Perhaps I should indeed make a new component.

File: frontend/src/components/Page/DefaultPage.jsx [hunk 4]
```diff
@@ -1,12 +1,36 @@
...
+
+    const theme = useBoundStore((state) => state.theme);
```
[the remark this answers is not in the document]
Comment (BeritJanssen) [reply frontend/src/components/Page/DefaultPage.jsx:11]: This should indeed still be adjusted.

**Pull request merged**